scripts/popup.js

```diff
@@ -0,0 +1,142 @@
+  },
+
+  {
+    id: '2',
+    img: 'images/works2/work2.png',
+    title: 'Multi-Post Stories',
+    tags: ['Canopy', 'Back End Dev', '2015'],
+    description: ' A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    categories: ['html', 'css', 'javascript'],
+    detail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    liveVersion: '#',
+    source: '#',
+  },
+
+  {
+    id: '3',
+    img: 'images/works2/work3.png',
+    title: 'Tonic',
+    tags: ['Canopy', 'Back End Dev', '2015'],
+    description: ' A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    categories: ['html', 'css', 'javascript'],
+    detail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    liveVersion: '#',
+    source: '#',
+  },
+
+  {
+    id: '4',
+    img: 'images/works2/work4.png',
+    title: 'Multi-Post Stories',
+    tags: ['Canopy', 'Back End Dev', '2015'],
+    description: ' A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    categories: ['html', 'css', 'javascript'],
+    detail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    liveVersion: '#',
+    source: '#',
+  },
+];
+const worksGridLayout = document.getElementById('worksGridLayout');
+function test(id) {
+  const popup = document.getElementById('popup');
+  const wrapper = document.getElementById('wrapper');
+  const html = `
+    <div class="workDetailContainer ">
+    <!-- body -->
+    <div class="workDetailBody">
+    <div class="workDetailBodyHeader">
+        <h1 class="worksBodytitle">${projects[id - 1].title}</h1>
+        <i class="" id="workDetailClose"><img src="./images/work-detail-cancel.png" alt="icon-cancel"></i>
+        </div>
+        <!-- description -->
+        <div class="frame2">
+            <span class="name">${projects[id - 1].tags[0]}</span>
+            <span class="dot"></span>
+            <span class="role">${projects[id - 1].tags[1]}</span>
+            <span class="dot"></span>
+            <span class="year">${projects[id - 1].tags[2]}</span>
```

Comment:
- [ ] You have nicely created your project object, but I have some suggestions to raise regarding the way you are exploring the data, let's focus on this `tags` array and category, in my opinion, they way you are rendering the tag is still static because when you will add a new project, it will require you to come and add index[n] on the new project you just added in the object you see, but the purpose of this milestone is to make that easy by automating the action, since tags is an array, you can map over its element and render them efficiently, see? 
```
  projects.map((project) => (
       project.id // for id
       
        project.title // for title etc...
        
         project.tags.map((tag) => <li> tag</li>))
        
          //same for the category as it is an array
        
        project.categories.map(() =>........
                    
))
``` 

- In this way, you will not encounter any issues in the future if you need to add a new project, in a scenario where you have 100 tags maybe, are you sure you do tags[0], tags[1].........tag[n]? I am sure you can't, so please update the syntax fusing the method I just showed you and request the review.